src/test/java/com/google/jenkins/plugins/computeengine/ComputeEngineCloudIT.java

```diff
@@ -283,18 +292,22 @@ public void testMultipleLabelsInConfig() throws Exception {
         // Add a new node
         Collection<NodeProvisioner.PlannedNode> planned = cloud.provision(new LabelAtom(LABEL), 1);
 
-        String provisionedLabels = planned.iterator().next().future.get().getLabelString();
+        String name = planned.iterator().next().displayName;
+
+        planned.iterator().next().future.get();
+
+        String provisionedLabels = r.jenkins.getNode(name).getLabelString();
         // There should be a planned node TODO
         assertEquals(logs(), MULTIPLE_LABEL, provisionedLabels);
     }
 
-    @Test(timeout = 300000)
+    @Test(timeout = 300000, expected = ExecutionException.class)
```

Comment:
My question is, why did this exception not get raised before this PR's change. What led to us needing to expect this exception now?

Reply:
We have simplified code to not call all exceptions..
https://github.com/jenkinsci/google-compute-engine-plugin/pull/26/files#diff-13ec31119ed683508c3eaf94108b0ea9R205

Reply:
That exception will be call in NodeProvisioner if it will be thrown..

Reply:
gotcha. I'll take a closer look when I get the chance and we can get this merged soon. I was on vacation last week.